ts/commands/DashboardCommands.ts

```diff
@@ -14,6 +14,7 @@ class DashboardCommands {
       .option('--username', 'Metricly Username')
       .option('--password', 'Metricly Password')
       .option('--profile', 'Metricly profile', /.*/, 'default')
+      .option('--format', 'output format: text, json', /.*/, 'text')
```

Comment:
You can define available strings in an array using a built in array validator: https://www.npmjs.com/package/caporal#array-validator

Reply:
do you know how to define the default with that approach?

Reply:
Not sure, though I'd either change the if statements into if/else when using it, or assign a default if the option is null right inside the initializer a couple logs down.

Reply:
done